lib/util/index.js

```diff
@@ -2,6 +2,7 @@
  * General Transitive utilities library
  */
 
+import isNumeric from 'isnumeric/isNumeric'
```

Comment:
I'm not a huge fan of adding a whole new package just for this little use here. I'd just go ahead and try to inline some kind of logic in this file.

Reply:
I removed the explicit reference in https://github.com/conveyal/transitive.js/pull/62/commits/7e65ad5f2d7327d5930cdae7a71d2c1045f7f8d8. 

Interestingly, `isnumeric` is implicitly referenced by another library (one of CSS libraries it looks like).